src/types.ts

```diff
@@ -38,6 +38,12 @@ export interface PluginOptions {
    * necessary.
    */
   parseNative?: boolean
+  /**
+    * Enable use of `tsconfck.resolveWithEmptyIfConfigNotFound`.
```

Comment:
Seems odd to mention an implementation detail in the description

Reply:
That's write, I'd rather copy the docs in tsconfck instead